platform/mellanox/mlnx-platform-api/sonic_platform/psu.py

```diff
@@ -304,7 +306,14 @@ def get_model(self):
         Returns:
             string: Model/part number of device
         """
-        return self.vpd_parser.get_model()
+        current_model = self.vpd_parser.get_model()
+        if current_model != 'N/A' and current_model != self.model and os.path.exists('/usr/share/sonic/platform/psu_sensors_conf_updater'):
+            subprocess.run("cp -f /etc/sensors.d/sensors.conf /tmp/sensors.conf.orig", shell=True)
```

Comment:
Suggest to use `utils.run_command`. For security reason, suggest to run the command with a list instead of string. Example:
```python
utils.run_command(['cp', '-f', '/etc/sensors.d/sensors.conf', '/tmp/sensors.conf.orig'])
```

Reply:
updated to use utils.run_command instead